plugins/c8y_configuration_plugin/src/topic.rs

```diff
@@ -1,26 +1,26 @@
 use mqtt_channel::TopicFilter;
 
 #[derive(Debug, Clone, Eq, PartialEq)]
-pub enum ResponseTopic {
-    ConfigSnapshotResponse,
-    ConfigSnapshotRequest,
-    ConfigUpdateRequest,
-    ConfigUpdateResponse,
+pub enum ConfigResponseTopic {
```

Comment:
The enum is named Config**Response**Topic but has variants named **Request**. Either change the name of the enum or separate the enum variants into separate `Request` and `Response` enums.